projects/storefrontapp/src/assets/qualtrics.js

```diff
@@ -0,0 +1,78 @@
+(function () {
```

Comment:
Reminder for myself to remove it.
Please ignore this comment until the final merge.